pkgs/development/python-modules/spyder/default.nix

```
@@ -41,14 +41,14 @@
 
 buildPythonPackage rec {
   pname = "spyder";
-  version = "5.5.1";
+  version = "5.5.2";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `setuptools` to `nativeBuildInputs`.